supplyChainSimulation/CapacityPlanning.de.resx

```diff
@@ -0,0 +1,196 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<root>
+  <!--
```

Comment:
Missing translations: Period -> Periode

Reply:
Braucht es in dem File die Übersetzung überhaupt? 
Auf der Seite kommt doch Periode gar nicht vor?

Reply:
Bitte führe das Programm einmal aus und schau es dir an.

Reply:
Braucht es die Übersetzung im dem View?

Reply:
Denke du meinst es bei der Einkaufsdisposition. Ist erledigt

Reply:
Ah mein Fehler, ja meine ich. Danke!